homeassistant/components/auth/__init__.py

```diff
@@ -168,6 +169,8 @@
 
 CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
 
+DELETE_CURRENT_TOKEN_DELAY = 2
```

Comment:
0.1 should be enough here, but there is always a risk the user's event loop is blocked, and a 2s delay to delete the token is fine